packages/liquid-html-parser/grammar/liquid-html.ohm

```diff
@@ -393,13 +393,18 @@ LiquidDoc <: Helpers {
   LiquidDocNode =
     | paramNode
     | exampleNode
+    | descriptionNode
     | fallbackNode
-
+    
   // By default, space matches new lines as well. We override it here to make writing rules easier.
   strictSpace = " " | "\t"
   // We use this as an escape hatch to stop matching TextNode and try again when one of these characters is encountered
   openControl:= "@" | end
 
+  descriptionNode = "@description" strictSpace* descriptionContent
+  descriptionContent = anyExceptStar<endOfDescription>
+  endOfDescription = strictSpace* openControl
```

Comment:
For the moment, this will include all leading and trailing whitespaces / new lines, so if you have:

```
@description asdf
@param
```

The content will match include `\n` - notice the empty second line:
```
@description asdf

```

This is because descriptions themselves can contain multiple lines, so we need to accept that within the content. We CAN, however strip trailing / leading whitespace / newlines, which is tracked [here](https://github.com/Shopify/developer-tools-team/issues/552)

That's something worth investigating separately, since this will also apply to the `@example` node, and we should consider edge cases.